src/quacc/atoms/skzcam.py

```diff
@@ -13,12 +13,489 @@
 from monty.io import zopen
 from monty.os.path import zpath
 
+
+class ElementInfo(TypedDict):
+    core: int
+    basis: str
+    ecp: str
+    ri_scf_basis: str
+    ri_cwft_basis: str
+
+
 if TYPE_CHECKING:
+    from ase.atom import Atom
     from numpy.typing import NDArray
 
 has_chemshell = find_spec("chemsh") is not None
 
 
+def create_orca_eint_blocks(
+    embedded_adsorbed_cluster: Atoms,
+    quantum_cluster_indices: list[int],
+    ecp_region_indices: list[int],
+    element_info: ElementInfo | None = None,
+    pal_nprocs_block: dict[str, int] | None = None,
+    method_block: dict[str, str] | None = None,
+    scf_block: dict[str, str] | None = None,
+    ecp_info: dict[str, str] | None = None,
+    include_cp: bool = True,
+    multiplicity: dict | None = None,
```

Comment:
For a lot of these type hints, the `str` key is actually not just any `str` but an element string. It would likely help the reader to know this. 

I'd recommend defining in the type hint block a new variable `ElementStr = Literal["H", "He", "Li", "Be", "B", "C", "N", "O", "F", "Ne", "Na", "Mg", "Al", "Si", "P", "S", "Cl", "Ar", "K", "Ca", "Sc", "Ti", "V", "Cr", "Mn", "Fe", "Co", "Ni", "Cu", "Zn", "Ga", "Ge", "As", "Se", "Br", "Kr", "Rb", "Sr", "Y", "Zr", "Nb", "Mo", "Tc", "Ru", "Rh", "Pd", "Ag", "Cd", "In", "Sn", "Sb", "Te", "I", "Xe", "Cs", "Ba", "La", "Ce", "Pr", "Nd", "Pm", "Sm", "Eu", "Gd", "Tb", "Dy", "Ho", "Er", "Tm", "Yb", "Lu", "Hf", "Ta", "W", "Re", "Os", "Ir", "Pt", "Au", "Hg", "Tl", "Pb", "Bi", "Po", "At", "Rn", "Fr", "Ra", "Ac", "Th", "Pa", "U", "Np", "Pu", "Am", "Cm", "Bk", "Cf", "Es", "Fm", "Md", "No", "Lr", "Rf", "Db", "Sg", "Bh", "Hs", "Mt", "Ds", "Rg", "Cn", "Nh", "Fl", "Mc", "Lv", "Ts", "Og"]` and then putting `ElementStr` in place of `str`  where appropriate.